server/routes/api/admin.js

```diff
@@ -0,0 +1,29 @@
+const express = require('express');
+const router = express.Router();
+
+// PLACEHOLDER ADMIN
+const Admin = require('./path/to/Admin');
```

Comment:
If you're waiting for another piece of code to be completed for something to work, it is nice (in my opinion) to leave a TODO comment or a note for other developers about it.